### PR TITLE
CD - Initial plan and resources - Fixes #5

### LINKED ISSUE #5: Define Webinar Agenda
Define an agenda for the Webinar series.

### PR DESCRIPTION
**Pull request by KietNhiTran**
* Added First week plan
* Added first top of mind
* Added Reference resources for further exploration and prepare

- Fixes #5

### CODE REVIEW

File: Continuous Delivery and Operation Excellence/Continuous Delivery.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Content gathered for Continuous Delivery
+
+## Delivery formats
```
Comment (PlagueHO) [inline Continuous Delivery and Operation Excellence/Continuous Delivery.md:3]: FYI, needs blank line after ## to be standards compliant MD. :grin:

File: Continuous Delivery and Operation Excellence/Continuous Delivery.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+# Content gathered for Continuous Delivery
+
+## Delivery formats
+* Webinar 
+* Self-service content
```
Comment (PlagueHO) [inline Continuous Delivery and Operation Excellence/Continuous Delivery.md:6]: What sort of self-service content? E.g. MS Learn Module, Documents, Videos?

File: Continuous Delivery and Operation Excellence/Continuous Delivery.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+* Lab
+
+## Plans
+Plan | Topic | Delivery Format 
+---- | ----- | ---------------
+Week 1 | Fast and Continuous Delivery in SaaS SDLC | Webinar
+
+## Plans Details
+### DevOps culture | Webinar | Kickoff for this item
+    - DevOps Cultures
```
Comment (PlagueHO) [inline Continuous Delivery and Operation Excellence/Continuous Delivery.md]: Could just be "DevOps Culture"

File: Continuous Delivery and Operation Excellence/Continuous Delivery.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    - Benefit of DevOps culture in related to SaaS Journey:
+        ○ Ref the Continuous Delivery section in ISV SaaS Playbook(Grow Your ISV Business with SaaS: Microsoft Practices Development Playbook)
+    - How DevOps and automation can improve productivities, security and fast go to market, A/B testing
```
Comment (PlagueHO) [inline Continuous Delivery and Operation Excellence/Continuous Delivery.md]: productivities -> productivity

File: Continuous Delivery and Operation Excellence/Continuous Delivery.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+    - Sharing resources
+    - Microsoft tools & Open source tools sharing
+    - Gather next interests
```
Comment (PlagueHO) [inline Continuous Delivery and Operation Excellence/Continuous Delivery.md]: Can you clarify this one?

File: Continuous Delivery and Operation Excellence/Continuous Delivery.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+    - Asking for input on interests on Office Hours, frequency 
+    - Asking for input of common challenges in speeding up SDLC
+
+## Top of minds & Reference resources
```
Comment (PlagueHO) [inline Continuous Delivery and Operation Excellence/Continuous Delivery.md:40]: Might also be worth having some external resources like DORA State of DevOps. 

Could also add links to the MS Learn DevOps courses.

**Comment by PlagueHO** [comment]
Great stuff @KietNhiTran.

One thing that might be useful is: "What is different/challenging in implementing DevOps in a multi-tenant SaaS environment". For example, multi-tenant monitoring, deployment rings, diagnostics & debugging considerations. I think where the gaps are because documentation is targeted mostly at SMC/Enterprise customers. ISVs face different/additional challenges, so we can define what those challenges are and how to solve them, I think ISVs will love us 😀

File: Continuous Delivery and Operation Excellence/Continuous Delivery.md (hunk 2 above)
Comment (KietNhiTran) [reply Continuous Delivery and Operation Excellence/Continuous Delivery.md:6]: It is just a list of planned formats. See details below.

File: Continuous Delivery and Operation Excellence/Continuous Delivery.md (hunk 3 above)
Comment (KietNhiTran) [reply Continuous Delivery and Operation Excellence/Continuous Delivery.md]: fixed

File: Continuous Delivery and Operation Excellence/Continuous Delivery.md (hunk 1 above)
Comment (KietNhiTran) [reply Continuous Delivery and Operation Excellence/Continuous Delivery.md:3]: Fixed

File: Continuous Delivery and Operation Excellence/Continuous Delivery.md (hunk 6 above)
Comment (KietNhiTran) [reply Continuous Delivery and Operation Excellence/Continuous Delivery.md:40]: Already there. But I think links is detached. Will fix

File: Continuous Delivery and Operation Excellence/Continuous Delivery.md (hunk 5 above)
Comment (KietNhiTran) [reply Continuous Delivery and Operation Excellence/Continuous Delivery.md]: It is the goal of the webinar.

**Comment by KietNhiTran** [comment]
Hi all,
This page at this moment is not for publishing. It is draft working page which I used to communicate and share idea. 
The published page is not there yet.

**Comment by PlagueHO** [comment]
@KietNhiTran - you could switch the PR to a draft PR if you want?

image

**Comment by PlagueHO** [comment]
I've added the remaining items for the plan.

File: Continuous Delivery and Operation Excellence/Continuous Delivery.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+Plan | Topic | Delivery Format 
+---- | ----- | ---------------
+Week 1 | Fast and Continuous Delivery in SaaS SDLC | Webinar
+Week 2 | Modernizing Compute | Webinar
+Week 3 | Application Storage Modernization | Webinar
+Week 4 | Innovation with Data & AI | Webinar
+Week 5 | Azure Marketplace | Webinar
+Week 6 | Identity in SaaS | Webinar
+Week 7 | The ISV Modernization Journey | Webinar
+Week 8 | Modern Architecture Patterns | Webinar
```
Comment (sajeetharan) [inline Continuous Delivery and Operation Excellence/Continuous Delivery.md]: These webinars are scheduled for the entire program right? should have have this in this file or at the main .md file with reference to devops content? @PlagueHO any thoughts?

File: Continuous Delivery and Operation Excellence/Continuous Delivery.md [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+Office Hours for Q&A on DevOps topics
+Office Hours for Q&A on Operation Excellence topics: Monitoring, Governance
+
+Challenges of streamline development processes of ISV:
+* A common platform for CI /CD
+* A pipeline, process which can be replicated and reuse across products and teams
+* Code qualities ensuring during development phrase
+* Infrastructure as Code habit and practices
+* Integration test and env for integration tests
+* TODO: add specific difficulties of ISV
```
Comment (sajeetharan) [inline Continuous Delivery and Operation Excellence/Continuous Delivery.md:51]: Adding some specific difficulties of ISVs 

-  Creating a custom pipeline
-  Monitoring and Error logs with Pipeline
-  Different tools for those who are not using AzDevops and Github ( Jenkins/ Gitlab / Hashicorp , Vagrant, Ansible etc )

File: Continuous Delivery and Operation Excellence/Continuous Delivery.md (hunk 7 above)
Comment (sajeetharan) [reply Continuous Delivery and Operation Excellence/Continuous Delivery.md]: Based on dev's feedback , we should move this as
Week 1 | The ISV Modernization Journey | Webinar
Week 2 | Modern Architecture Patterns | Webinar

Comment (KietNhiTran) [reply Continuous Delivery and Operation Excellence/Continuous Delivery.md]: Agree!

**Comment by KietNhiTran** [comment]
Closed for now since it is planning and thinking, not the real content to public. Will provide a PR for DevOps content soon.

**Pull request closed without merging**